src/pages/Quiz/QuestionWrapper/index.js

```diff
@@ -0,0 +1,164 @@
+import React, { useEffect, useState } from 'react';
+
+import { Typography, Paper, Divider, Button, Box } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import Question from '../../../components/Question';
+
+import Score from '../../../components/Score';
+
+import {
+  fetchSelectedQuestions,
+  fetchTestDetails,
+} from '../../../actions/quiz.js';
+
+import SkipModalWindow from '../../../components/SkipModalWindow';
+
+const useStyles = makeStyles((theme) => ({
+  paper: {
+    maxWidth: '930px',
+    padding: '10px',
+    margin: '0 auto',
+  },
+  quizNavigation: {
+    width: '100%',
+    margin: '10px 0 0 auto',
+    display: 'flex',
+    justifyContent: 'flex-end',
+  },
+}));
+
+const mapStateToProps = (props) => ({
+  testTitle: props.quiz.selectedTest.title,
+  selectedQuestion: props.quiz.selectedQuestion,
+});
+
+const mapDispatchToProps = (dispatch) =>
+  bindActionCreators(
+    {
+      fetchTestDetails: fetchTestDetails,
+      fetchSelectedQuestions: fetchSelectedQuestions,
+    },
+    dispatch,
+  );
+
+const QuestionWrapper = (props) => {
+  const {
+    match: { params: { testNumber = null } = {} } = {},
+    selectedQuestion = null,
+    testTitle = null,
+  } = props;
+
+  const [active, setActive] = useState(false);
+  const [testAnswers, setTestAnswers] = useState([]);
+  const [selectedAnswer, setSelectedAnswer] = useState({
+    userAnswer: null,
+    correctAnswer: null,
+  });
+
+  const classes = useStyles();
+
+  useEffect(() => {
+    props.fetchTestDetails(testNumber);
+    props.fetchSelectedQuestions(testAnswers.length);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [testNumber, testAnswers]);
+
+  if (selectedQuestion === null) {
+    if (testAnswers.length) {
+      return (
+        <Score
+          testTitle={testTitle}
+          score={testAnswers.filter((answer) => answer).length}
```

Comment:
you can use an easier syntax for filtering

instead of 

```jsx
score={testAnswers.filter((answer) => answer).length}
```

you can use

```jsx
score={testAnswers.filter(Boolean).length}
```